client/src/pages/home.tsx

```diff
@@ -27,18 +29,23 @@ export const HomePage: FC = () => {
   useEffect(() => {
     const loadData = async () => {
       if (file) {
-        const csvData = await readCsv(file);
-        setData(csvData);
+        if (token === undefined) {
```

Comment:
Why do they need a token if there be invited by url. Participants don't want to sign up to protect there anatomy

Reply:
Ah makes sense. I'll remove this.